LottoDairy/TabBarScene/TabBarController.swift

```diff
@@ -52,8 +54,14 @@ extension TabBarController: UITabBarControllerDelegate {
     
     func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
         guard let controller = viewControllers?[selectedIndex] as? UINavigationController else { return }
-        if selectedIndex == 1 {
+
+        switch selectedIndex {
+        case 0:
+            onCalendarFlowSelect?(controller)
+        case 1:
             onHomeFlowSelect?(controller)
+        default:
+            controller
```

Comment:
```suggestion
        switch TabBarComponents(rawValue: selectedIndex) {
        case .calendar:
            onCalendarFlowSelect?(controller)
        case .home:
            onHomeFlowSelect?(controller)
        default:
            controller
        }
```

selectedIndex를 switch문으로 분기하는 대신 `TabBarComponents`를 사용한 분기처리가 코드 가독성에 더 좋을 것 같습니다!

Reply:
`enum`타입을 이용해서 수정했습니다!